dotcom-rendering/src/components/DropCap.tsx

```diff
@@ -12,19 +12,22 @@ const dropCap = css`
 	/* stylelint-disable-next-line property-disallowed-list -- we’re setting custom line height and font weight */
 	font-family: ${fonts.headline};
 	float: left;
-	font-size: 111px;
-	line-height: 92px;
+	font-size: 7rem;
+	line-height: 5.75rem;
```

Comment:
Is the move to `rem` resulting from #10860? Should we take the opportunity to change all the other usage of `px` in this file at once?